sdk/cosmos/sdk/src/main/java/com/azure/data/cosmos/sync/CosmosSyncItem.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.cosmos.sync;
+
+import com.azure.data.cosmos.CosmosClientException;
+import com.azure.data.cosmos.CosmosItem;
+import com.azure.data.cosmos.CosmosItemRequestOptions;
+
+/**
+ * The Cosmos synchronous item.
+ */
+public class CosmosSyncItem {
+    private CosmosSyncContainer container;
+    private CosmosItem asyncItem;
+    private String id;
+    private Object partitionKey;
+
```

Comment:
We should make these final, if they are only initialized during construction and never plan to change throughout the course of this instance.

Reply:
Done